### PR TITLE
Add config file path to eslint package.json

### PR DESCRIPTION
**Pull request by bipuladh**
Signed-off-by: Bipul Adhikari <badhikar@redhat.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -135,7 +135,7 @@
     "node": ">=14.16.0"
   },
   "lint-staged": {
-    "*.{ts,tsx}": "eslint --cache --fix",
+    "*.{ts,tsx}": "eslint --cache --fix -c ./.eslintrc.js",
```
Comment (SanjalKatiyar) [inline package.json:138]: we don't have to give path for stylelint and prettier config files as well ??

Comment (bipuladh) [reply package.json:138]: for some reason eslint is not detecting it. For prettier and stylelint I don't see any problems as of now.

Comment (SanjalKatiyar) [reply package.json:138]: ack...

**Pull request merged**